### PR TITLE
Replace LMS docker image with new community version (#381)

### LINKED ISSUE #381: Update Logitech Media Server to community docker image (Lyrion Media Server)
With the discontinuation by Logitech, the project has moved into a community space https://lyrion.org/. The currently used docker image within the Rockon has not been updated to pull the latest version from a new location (in fact, while it had automated build pipeline, the underlying docker development has not been updated in a few years, because it was working and updating fine).

the LMS community is publishing a new official docker image: https://hub.docker.com/r/lmscommunity/logitechmediaserver

the current LMS Rockon should be adapted to take advantage of the new actively developed docker image (will likely require a few parameters to be adjusted.

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #381.

Changes performed:
- Replace Docker Image with new community version (renamed to Lyrion Media Server after Logitech shut project down).
- Update Parameter Changes/Description
- Formatting

### Information on docker image
- new docker image: [<link to image page on docker hub>](https://hub.docker.com/r/lmscommunity/logitechmediaserver/)
- is an official docker image available for this project?: Yes - project migrated from Logitech to community: https://lyrion.org/

### Checklist
- [X] Passes [JSONlint](https://jsonlint.com) validation
- [ ] Entry added to `root.json` in _alphabetical_ order (for new rock-on only)
- [X] `"description"` object lists and links to the docker image used
- [X] `"description"` object provides information on the image's particularities (advantage over another existing rock-on for the same project, for instance)
- [X] `"website"` object links to project's main website

### CODE REVIEW

**Comment by Hooverdan96** [comment]
Sample Installation performed using local Rockon json file:

image

Define Shares:
image

Define Ports:
image

Define UID/GID under which container is running (this is new compared to the previous version's running always under root)
image

Summary before install:
image

LMS Server Screen:
image

After a bit of scanning and configuration, the server is now streaming (again) to a PiCorePlayer on the same network.

I attempted to migrate the settings and libray files, but that proved too difficult considering the changes since the last version update, and actual path changes in the community image. It continued to cause a restart of the container due to fatal errors. Once I abandoned the previous LMS config files, it started and remained stable - downside was that I had to reconfigure quite a few settings, but having documented those prior to using the new image made it fairly easy and straightforward. I didn't spend time online trying to find out whether/how others have migrated, but there might be some guides out there.

File: lms_latest.json [hunk 1]
```diff
@@ -1,54 +1,70 @@
...
+                "ports": {
+                    "9000": {
+                        "description": "WebUI port. This MUST be set to 9000",
+                        "host_default": 9000,
+                        "label": "WebUI port",
+                        "protocol": "tcp",
+                        "ui": true
+                    },
+                    "9090": {
+                        "description": "Command Line Interface (used for example by Android app Squeezer). Suggested Default: 9090",
```
Comment (phillxnet) [inline lms_latest.json:30]:
```suggestion
                        "description": "Command Line Interface (used for example by Android app Squeezer). This MUST be set to 9090",
```
@Hooverdan96 Could you check my suggestion here: taken from: https://hub.docker.com/r/lmscommunity/logitechmediaserver

> Please note that both the http port 9000 and the cli port 9090 must always be mapped 1:1. 

There is more on this at the given reference though.

**Review by phillxnet — CHANGES_REQUESTED** [review]
@Hooverdan96 Thanks for seeing to this update. My suggestion is likely more cosmetic given the use of `--net host` by this Rock-on. But this way we surface to the use the intended port.

File: lms_latest.json (hunk 1 above)
Comment (Hooverdan96) [reply lms_latest.json:30]: yes, good point. There are ways to changes these (when digging into the documentation a bit more), but it's like a three-way shuffle to do so. I will commit your suggestion.

**Review by phillxnet — APPROVED** [review]

File: lms_latest.json [hunk 2]
```diff
@@ -1,54 +1,70 @@
...
+                        "label": "Media Storage"
+                    },
+                    "/playlist": {
+                        "description": "Choose a Share for LMS Playlists. E.g.: create a Share called lms-playlist for this purpose alone. You can also assign other media Shares on the system after installation.",
+                        "label": "Playlist Storage"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid User ID (UID) to run Home Assistant with. It must have full permissions to all Shares mapped in the previous step.",
```
[the remark this answers is not in the document]
Comment (Hooverdan96) [reply lms_latest.json:57]: oops. Thanks for catching this. I guess, this was also wrong in the original LMS Rockon (I guess, I based it on the HomeAssistant one and did not correct this ...

**Review by phillxnet — APPROVED** [review]

**Comment by phillxnet** [comment]
@Hooverdan96 My apologies. It looks like I dropped the ball on following this one up after that last minor change. @alexjulien PR (#384) prompted me to do a final test on this PR prior to merge and publish:

On a Leap 15.6 host running 5.0.14-0 we have the pre-install summary of:

## Pre-install summary
lms-pre-install-summary

## Installed
lms-installed

## lms
lms-settings-in-webui

I'm surprised they have maintained the Logiteck branding there !!

But this looks to be working as you originally presented and we have our cosmetic Rock-on elements all sorted now hopefully.

I'll merge and publish.

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**